src/main/java/net/fabricmc/mappingio/adapter/ForwardingMappingVisitor.java

```diff
@@ -27,6 +27,12 @@
 import net.fabricmc.mappingio.MappingFlag;
 import net.fabricmc.mappingio.MappingVisitor;
 
+/**
+ * A mapping visitor that forwards all visit calls to another {@link MappingVisitor}.
+ *
+ * <p>Does nothing useful by itself, it's intended to be extended.
```

Comment:
> it's intended to be extended

I feel that's obvious since the class is abstract.

Reply:
True, I didn't notice it being abstract 😅